pkgs/development/python-modules/pandas/default.nix

```diff
@@ -110,6 +110,12 @@ buildPythonPackage rec {
     "test_clipboard"
   ];
 
+  disabledTestPaths = [
+    # some test in this file either crashes python or hangs for hours
+    # TODO: figure out why and how to fix it
+    "tests/io/pytables/test_pytables_missing.py"
```

Comment:
Is this still required with hdf5 1.10?

Reply:
You are right it's not needed :+1: 
It was a leftover of my tests :disappointed:

Reply:
Nothing to worry about. Sometimes you just need a second pair of eyes.